ui/jasmine_test/spec/helpers/stubs/StubAnalyser.js

```diff
@@ -0,0 +1,8 @@
+export class StubAnalyser {
```

Comment:
nit: add copyright at the top of each new file

Reply:
good catch